src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -22,12 +22,17 @@ export class NoticeOfDisputeService {
   RepresentedByLawyer = DisputeRepresentedByLawyer;
   RequestTimeToPay = DisputeCountRequestTimeToPay;
   RequestReduction = DisputeCountRequestReduction;
-  RequestCourtAppearance = DisputeCountRequestCourtAppearance;
+  RequestCourtAppearance = DisputeRequestCourtAppearanceYn;
   PleaCode = DisputeCountPleaCode;
+  ContactType = DisputeContactTypeCd;
 
   noticeOfDisputeFormConfigs: NoticeOfDisputeFormConfigs = {
     disputant_surname: { value: null, options: { validators: [Validators.required] } },
     disputant_given_names: { value: null, options: { validators: [Validators.required] } },
+    contact_given_names: { value: null },
+    contact_surname:{ value: null },
+    contact_law_firm_name: { value: null },
+    contact_type: { value: this.ContactType.Individual, options: { validators: [Validators.required] } },
```

Comment:
remove `work_phone_number`
I saw you removed in translation, I suppose that is not needed

Reply:
done